pages/tidal tools/machine_stats.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Gather Machine Stats
+keywords:
+last_updated: February, 2021
+summary: "Gather machine stats from remote environments"
+sidebar: main_sidebar
+permalink: machine_stats.html
+folder: tidaltools
+---
+
+## Overview
+
+Getting an accurate view of your infrastructure needs is useful when planning a
+cloud migration.  Most datacenter operations groups have a good view of their
+overall storage utilization from various SAN and virtualization management
+tools, but relying on these aggregated data points often causes teams to
+underestimate the storage needs of their applications in the cloud.
+
+When taking an _application-centric_ approach to cloud migration, getting the
+resource utilization from each individual server gives you a more accurate view
+of each application's resource requirements in the cloud and ignores the
+trickery of thin-provisioning from SAN tools.  This allows you to confidently
+plan data replication time, or other migration metrics on an app-by-app basis.
+
+{% include tip.html content="As your cloud migration will likely take place
+over many months or years, it's important to have current resource requirements
+throughout your journey. To accomplish this, setup `machine_stats` in a cron
+job or Scheduled Task and synchronize your data on a _daily_ basis to Tidal
+Migrations." %}
+
+## Windows
+
+### Introduction
+
+This is a simple and effective way to gather machine stats (RAM, Storage,
+CPU) from a Windows Server environment. It uses
+[WinRM](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management)
+to `Invoke-Command` across your servers, creating a JSON file to securely send
+to your Tidal Migrations instance using the tidal command.
+
+### Requirements and Dependencies
+
+Prior to the steps on syncing your hypervisors, be sure to have the below
+dependencies setup:
+
+- To get started you will need to have Tidal Tools installed. You can check out
+  this [article](tidal-tools.html) on how to install it.
```

Comment:
QQ, is it an article or a guide? I guess something times we use those words interchangeably. For me, `guide` sounds more appropriate. What do you think?

Reply:
Good catch! Yes, let's stick with “guide”!

Reply:
FYI, we have the same on line 94